src/stories/color_palette/__test__/colors.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { ColorPalette } from "../colors";
+
+type Story = StoryObj<typeof ColorPalette>;
+
+export const ColorsPaletteStory = (args: Story["args"]) => (
+  <ColorPalette {...args} />
+);
+
+export default {
+  title: "Color Palette",
+  component: ColorPalette,
+} as Meta;
```

Comment:
genre ça

Reply:
En fait c'est Storybook qui le demande pour pouvoir l'afficher. Tu peux allez tester si tu veux, suffit de clone le repo et de faire un npm run storybook. Tu verras facilement comment ça marche
![Screenshot from 2024-04-30 15-34-53](https://github.com/The-Geneps-Personnal-Project/EsportsStats-FrontWeb/assets/71601150/01d59341-0bf5-44e8-a12c-6f878fddf575)

Reply:
kk nice